service/service.go

```diff
@@ -146,3 +146,19 @@ func New(options ...Option) (Owner, error) {
 
 	return svc, nil
 }
+
+type niladicStart func()
```

Comment:
Got it from https://golang.org/pkg/sync/#Once.Do
Niladic - zero arguments,
Monadic - one argument,
Dyadic...